hca/config.py

```diff
@@ -37,6 +36,7 @@ class ProgressBarStreamHandler(object):
     """
     @staticmethod
     def write(msg):
+        import tqdm  # imported here to avoid unnecessary dependency breaks
```

Comment:
tqdm is in the requirements.txt, but it looks like only the requirements-dev.txt is installed by the pipeline (https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/data-store/-/jobs/54102). shouldn't the pipeline do a `pip install -r requirements.txt` before running `python setup.py`, which would resolve this problem?

Reply:
`setup.py` should not have outside dependencies.